### PR TITLE
Report v2: add back debugString

### PR DESCRIPTION
**Pull request by ebidel**
screen shot 2017-04-24 at 1 44 31 pm

### CODE REVIEW

File: lighthouse-core/report/v2/report-styles.css [hunk 1]
```diff
@@ -17,6 +17,7 @@
  :root {
   --text-font-family: '.SFNSDisplay-Regular', 'Helvetica Neue', 'Lucida Grande', sans-serif;
   --body-font-size: 13px;
+  --body-line-height: 20px;
```
Comment (patrickhulce) [inline lighthouse-core/report/v2/report-styles.css]: is this used?

Comment (ebidel) [inline lighthouse-core/report/v2/report-styles.css]: In 47. Must have missed it when porting. Gives us a bit more breathing room.

Comment (paulirish) [inline lighthouse-core/report/v2/report-styles.css]: I always see and prefer line-height specified as unitless: https://www.google.com/search?q=line-height+unitless

seems a bit better.

Comment (ebidel) [inline lighthouse-core/report/v2/report-styles.css]: done

**Comment by ebidel** [comment]
Finally passing Travis. Merging.

**Pull request merged**